### ISSUE TITLE
Do not remove screenshots unless uploaded succesfully

### ISSUE THREAD

**Issue by jlsjonas** [mileswd/mac2imgur#102]
Given the recent issues ( #100 ) I've noticed my screenshots would still get deleted, physically removing the last available copy. delete on upload should only delete on successfull upload

**Comment by mileswd** [comment]
I agree - this used to be the case in our previous implementation, but since we moved to ImgurSession this was no longer possible. I'm happy to revisit this issue after we get a solution to #100.

Also, the screenshot is not deleted, but moved to `Trash` instead.

**Comment by jlsjonas** [comment]
*checks trash* o.O
well as was already suggested in #100 it might be time to move away from imgurSession
